frontend/src/components/car-list-item/car-list-item.tsx

```diff
@@ -37,6 +42,15 @@ const CarListItem: React.FC<CarListItemProps> = (props) => {
   const [modelFuelType, setModelFuelType] = useState<string>();
   const [modelTransmission, setModelTransmission] = useState<string>();
 
+  const { likedCars, handleLikeClick } = useContext(WishlistContext);
+  const isLiked = likedCars?.includes(model_id);
+
+  const likeClick = (event?: React.MouseEvent): void => {
+    event?.stopPropagation();
+    const data: WishlistInput = { modelId: model_id };
+    handleLikeClick(data);
+  };
```

Comment:
Where do you get the `model_id`? And why do we have variables in `snake_case` in code?

Reply:
We take this data from props, it comes to props in the current form from the backend.